week2/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang = "en">
+    <head>
+        <meta charset="utf-8">
+        <title>My first HTML Page</title>
+      </head>
+      <body>
+        <header>
+            <nav>
+                <a href="pages/about.html">about</a>
+            </nav>
+        </header>
+        <main>
+        <h1>My First HTML Page</h1>
+        <h2><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element"
+            title = "The best place to find the knowledge related to HTML"
+            target="_blank"
+            >HTML Elements</a></h2>
+        <p>HTML elements <i>usually</i> consist of opening and closing tags and content. </p>
```

Comment:
&lt;i&gt; is just used to make the text italic. Anything about styling should be in CSS. Here, we are emphasizing on "usually", hence we use &lt;em&gt; tag.